webapp/pages/profile/_id/_slug.vue

```diff
@@ -365,6 +362,12 @@ export default {
         fetchPolicy: 'cache-and-network',
       })
     },
+    setActivePosts() {
+      if (!this.Post) {
+        return []
+      }
+      return this.uniq(this.Post.filter(post => !post.deleted))
+    },
```

Comment:
could you explain what motivated these changes @Tirokk 
sorry, it's not obvious to me

Reply:
This code is probably written by @roschaefer .

Here I rewrote it a little to get the delete Post on the User Profile working again. But nothing serious I think.
He filters the deleted Posts out of the list array I think.